packages/EasyCore/src/Helpers/RecursiveStringsTrimmer.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Helpers;
+
+final class RecursiveStringsTrimmer implements StringsTrimmerInterface
+{
+    /**
+     * A list of array keys whose values will be ignored during processing.
+     *
+     * @var string[]
+     */
+    private $except;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function trim($data, ?array $except = null)
+    {
+        $this->except = $except ?? [];
+
+        if (\is_array($data)) {
+            return $this->cleanArray($data, '');
+        }
+
+        return $this->transform($data, '');
+    }
+
+    /**
+     * @param mixed[] $data
+     *
+     * @return mixed[]
+     */
+    private function cleanArray(array $data, string $keyPrefix = ''): array
```

Comment:
Shouldn't the second parameter be `nullable`?

Reply:
Fixed.